README.md

```diff
@@ -30,3 +30,25 @@ that belong in the OEM partition. Any RDEPENDS from these packages should
 be copied to the RDEPENDS in `board-packages` to ensure they are built.
 
 `coreos-base/coreos-oem-*` are metapackages for OEM specific ACIs. 
+
+# Updating
+
+To update follow the following steps:
+
+* Remove or rename the whole folder of the package to prepare the import from
+  upstream Gentoo, not only resetting the ebuild file but also any additional
+  files like patches or downstream additions under `files`.
+* Run `~/trunk/src/scripts/update_ebuilds --portage_stable . CATEGORY/PACKAGE`
+  in the `coreos-overlay` folder to import a new version from upstream Gentoo.
+* Commit the changes with a message like `CATEGORY/PACKAGE: Sync from Gentoo`.
+* Now find all downstream patches for the package by running
+  `git log CATEGORY/PACKAGE`. If everybody followed the process of resetting
+  before importing an upstream update, you only have to look for the commits
+  after the last update and port them to the new version. Otherwise you have
+  to compare the files manually to their upstream versions from older
+  [portage](https://github.com/gentoo/portage/) revisions.
+* Combine all old and new downstream patches into a single new commit with the
+  message `CATEGORY/PACKAGE: Apply Flatcar patches` to keep the number of
+  commits to port low. Make sure that you explain the changes and carry the
+  explanations from old commits over, either in the commit message or even
+  better through comments in the ebuild file.
```

Comment:
So far, I haven't followed that - I was mostly splitting the commits into smaller changes. But I think I'd be fine with having a single commit. About the explanations: comments in an ebuild are fine, but sometimes we drop a bunch of lines from an ebuild and a comment might look out of place. So it would be good to have a `README.md` file in the directory that explains the changes. So this file would need to be also updated/copied over every time we do an update, so it does not become stale.

Currently I'm working on adding the `README.md` files to each directory.

Reply:
I found having the explanations in the commit message helps useful but having a them in a README also works.